test/packaging/ansible/roles/setup-config/tasks/config-Win32NT.yaml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: create newrelic-infra directory structure
+  win_file:
+    path: C:\Program Files\New Relic\newrelic-infra
+    state: directory
+
+- name: install infra-agent configuration file
+  template:
+    src: ../files/etc/newrelic-infra.yml.j2
+    dest: C:\Program Files\New Relic\newrelic-infra\newrelic-infra.yml
+    owner: root
```

Comment:
Just curious, how does root work for windows?

Reply:
good catch! that is a leftover, I'll remove it